src/main/java/org/logstash/beats/BeatsHandler.java

```diff
@@ -81,19 +81,37 @@ public void exceptionCaught(ChannelHandlerContext ctx, Throwable cause) throws E
             if (!(cause instanceof SSLHandshakeException)) {
                 messageListener.onException(ctx, cause);
             }
-            final Throwable realCause = extractCause(cause, 0);
-            if (logger.isDebugEnabled()){
-                logger.debug(format("Handling exception: " + cause + " (caused by: " + realCause + ")"), cause);
+            if (silentException(cause)) {
+                if (logger.isDebugEnabled()) {
+                    logger.info(format("closing"), cause);
+                } else {
+                    logger.info(format("closing (" + cause.getMessage() + ")"));
+                }
             } else {
-                logger.info(format("Handling exception: " + cause + " (caused by: " + realCause + ")"));
+                final Throwable realCause = extractCause(cause, 0);
+                if (logger.isDebugEnabled()){
+                    logger.debug(format("Handling exception: " + cause + " (caused by: " + realCause + ")"), cause);
```

Comment:
Do you think we should also change this to `info` if `debug` is enabled on the logger to be consistent with the above log message? I know this was what we had before, but it seems a little unusual to change the log level of a logging statement just because the amount of detail in the message has changed

Reply:
makes sense, thanks